config/memory_statistics.py

```diff
@@ -0,0 +1,114 @@
+import click
+from swsscommon.swsscommon import ConfigDBConnector
+# from utilities_common.cli import AbbreviationGroup
```

Comment:
If not needed, please remove it.

Reply:
Thankyou for reviewing. I just removed it